.github/workflows/ci.yaml

```diff
@@ -37,10 +37,10 @@ jobs:
         os: ["ubuntu-latest"]
         pydantic-version: [">1"]
         python-version:
-          - "3.9"
```

Comment:
Are we definitely dropping 3.9  now? What's stopping us from going 3.12? (I thought it was AmberTools but that might not be the case with these changes?)

Reply:
If we are are following https://scientific-python.org/specs/spec-0000/ then we are good to drop it. 

That's a good question about 3.12, I'll open a PR and test if we can solve a 3.13 env.

Reply:
testing here https://github.com/OpenFreeEnergy/openfe/pull/850

Reply:
looks like pymbar 3.x needs a py 3.12 build https://github.com/OpenFreeEnergy/openfe/actions/runs/9033924083/job/24825299591?pr=850

I can do this, but given https://github.com/OpenFreeEnergy/openfe/issues/833 and https://github.com/choderalab/pymbar/issues/419#issuecomment-1955191199 I think we can move to pymbar 4 soon, thoughts?

Reply:
Let's aim to get pymbar support done before the next release and temporarily shorten our support range.

Reply:
Ah wait so it did work out in the end?